### ISSUE TITLE
Use aria-label to distinguish different nav menus

### ISSUE THREAD

**Issue by mmazanec22** [ctoec/ecis-experimental#1259]
From Deque: "Multiple instances of the same type of landmark SHOULD be distinguishable by different discernible labels (aria-label or aria-labelledby)."
